src/github.com/mozilla-services/pushgo/simplepush/handlers_endpoint.go

```diff
@@ -330,9 +330,11 @@ func (h *EndpointHandler) deliver(cn http.CloseNotifier, uaid, chid string,
 	worker, workerConnected := h.app.GetWorker(uaid)
 	var routingTime time.Duration
 
+	shouldRoute := h.alwaysRoute || !workerConnected
```

Comment:
Nit: We're only using shouldRoute once, and we're defining the condition before the comment where we actually use it. Might be a bit clearer to move the comment before the condition definition.